### DISCUSSION TITLE
请问下纯本文的抽取式阅读理解mrc用哪个模型合适？

### DISCUSSION THREAD

**Discussion by jyjy007** [PaddlePaddle/PaddleNLP#3812] [Q&A]
在paddlenlp下找到ernie-doc有支持mrc的模型，然后dureader有robust抽取式阅读理解数据集，但是似乎都没有现在封装的uie/doc-promt等模型用taskflow封装后简单上手。请问下目前纯文本问答抽取mrc这块用哪个模型能快速上手？

**Comment by wawltor** [comment]
目前我们对MRC的抽取工作建议是使用UIE来使用，ernie-doc等模型支持MRC的抽取，但是它的零样本效果不好，不太适合放入到Taskflow里面去

**Reply by jyjy007** [reply]
> 目前我们对MRC的抽取工作建议是使用UIE来使用，ernie-doc等模型支持MRC的抽取，但是它的零样本效果不好，不太适合放入到Taskflow里面去

uie加载模型时需要定义schema，对于开放性问题，query是不可枚举的，难道每次回答问题都要重新定义schema重新加载一次模型？这样效率很低哦。
